app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/Layout.scala.html

```diff
@@ -66,6 +69,18 @@
         <script src='@_root_.controllers.routes.Assets.versioned("javascripts/application.js")'></script>
     }
 
+    @hmrcStandardHeader = {
+
+        @hmrcHeader(Header(
+            homepageUrl = viewConfig.govUkUrl,
```

Comment:
This points to `"https://www.gov.uk/"` - are we saying that this should be the home-page?

Reply:
will be handled by JIRA-714